cmd/minishift/cmd/addon/addon.go

```diff
@@ -28,9 +28,10 @@ const (
 )
 
 var AddonsCmd = &cobra.Command{
-	Use:   "addons SUBCOMMAND [flags]",
-	Short: "Manages Minishift add-ons",
-	Long:  "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Use:     "addons SUBCOMMAND [flags]",
+	Aliases: []string{"addon"},
```

Comment:
resources are supposed to be plural for an API. I believe I made the hostfolders singular... but not happy with this. I'd rather make plural the default case and add singular as alias.